src/diffusers/__init__.py

```diff
@@ -96,6 +96,8 @@
 else:
     _import_structure["quantizers.quantization_config"].append("TorchAoConfig")
 
+_import_structure["quantizers.quantization_config"].append("FinegrainedFP8Config")
```

Comment:
We haven't moved to setting torch and accelerate as required deps yet (will happen soon). So this would have to go under a `is_torch_available() and not is_accelerate_available()` type guard.